test/ejabberd_tests/tests/cluster_commands_SUITE.erl

```diff
@@ -308,14 +403,21 @@ normalize_args(Args) ->
 
 %% Long timeout for mnesia and ejabberd app restart
 run_interactive(Cmd, Response) ->
-    run_interactive(Cmd, Response, timer:seconds(30)).
+    run_interactive(Cmd, Response, timer:seconds(60)).
 
 run_interactive(Cmd, Response, Timeout) ->
     Port = erlang:open_port({spawn, Cmd}, [exit_status]),
     %% respond to interactive question (yes/no)
     Port ! {self(), {command, Response}},
     loop(Port, [], Timeout).
 
+run(Cmd) ->
```

Comment:
yes we can, I will also reuse `normalize_args/1` and `loop/3` from the `ejabberdctl_helper` module